### PR TITLE
Adds the Beach Bum Language

### PR DESCRIPTION
**Pull request by ghost**
:cl: That Really Good Soda Flavor
add: People who are high and beach bums can now talk in their own stoner language.
tweak: Beach bums can only communicate with other beach bums and people who are high.
/:cl:

Radical, dude!

# If stupid lizards should get a language, then beach bums and high people definitely deserve one, too!

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
cowabunga

**Comment by kevinz000** [comment]
if ckey == "mrstonedone" default_language = STONER :^)

**Comment by Really-Good-Soda-Flavor** [comment]
Example

**Comment by Expletive** [comment]
So you're saying that if you want to murder someone, shoot them full of Nuka Cola and then they won't be able to shout for help in a language anyone can understand?

**Comment by Really-Good-Soda-Flavor** [comment]
Give them a lit rollie and wait a few seconds.

**Comment by Jordie0608** [comment]
When are we getting space-bogan language?

**Comment by ShadowDeath6** [comment]
does this force them to speak it over every other language or does it just remove common

if they can still speak lizard or whatever then there's not really a "muting" feature of it, which probably is for the best

**Comment by Really-Good-Soda-Flavor** [comment]
It only removes common for actual beach bums.

**Comment by octareenroon91** [comment]
At present the mutation explicitly removes common.

**Comment by Really-Good-Soda-Flavor** [comment]
> At present the mutation explicitly removes common.
> It only removes common for **actual beach bums.**

FTFY

**Comment by Cruix** [comment]
Can't wait for revhead bartenders/botanists to start a stoner revolution shouting where the heads are over common chat.

**Comment by ShadowDeath6** [comment]
it should remove common regardless of who has it, makes no sense that a regular crewmember can be drugged off their ass and still speak normally(albeit slurred) while beach bums who aren't drugged at all can't speak common

**Comment by ShizCalev** [comment]
can we add a little more bill and ted shit to this

http://www.billandted.org/howtospeak.htm

**Comment by octareenroon91** [comment]
>>At present the mutation explicitly removes common.

>> It only removes common for actual beach bums.

>FTFY

I see now I misread: the _mutation_ absolutely does remove common, but only beach bums get it. I was reading the drugginess procs as bestowing the mutation, rather than the language.

**Comment by FantasticFwoosh** [comment]
I had this idea a while ago, nice to see it in practice.

Duuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuudeee...

**Comment by YakumoChen** [comment]
yeah this seems like a rather bad buff to botany / anyone who is getting people high since it effectively acts as mute toxin

File: code/modules/language/beachbum.dm [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/datum/language/beachbum
+	name = "Beach Bumbian"
+	desc = "For when you're too toasted to speak normally."
+	speech_verb = "mumbles"
+	ask_verb = "grills"
+	exclaim_verb = "hollers"
+	key = "u"
+	space_chance = 80
+	default_priority = 1000 //Haha!
```
Comment (Profakos) [inline code/modules/language/beachbum.dm]: Would this mean that the curators will speak this ancient and forbidden tongue as their default language?

**Comment by Niknakflak** [comment]
Dude, this PR is like totally radical man

**Comment by Repukan** [comment]
Dude weed lmao

**Comment by Kromgar** [comment]
So beach bums will be an even more worthless role only able to speak to eachother wew

**Comment by XDTM** [comment]
Feels like it should be heavily distorted but not outright cryptic

**Comment by Really-Good-Soda-Flavor** [comment]
I've lowered the priority so it will not be used as a mute toxin.

**Comment by octareenroon91** [comment]
>So beach bums will be an even more worthless role only able to speak to eachother wew

Actually, making it default but not the only language on beach bums would match what ashwalkers get. Beach bumming is pretty rough as it is.

**Comment by Really-Good-Soda-Flavor** [comment]
They can still talk to people who are high, actually.

**Comment by Armhulen** [comment]
Yeah, smoke weed and you'll understand what the bums are saying, which is fucking awesome

**Comment by Armhulen** [comment]
give the name itself something new though, it makes no sense

**Comment by Really-Good-Soda-Flavor** [comment]
# Beachtongue

**Comment by optimumtact** [comment]
Hang ten brother

**Pull request merged**